be/src/exec/vectorized/cross_join_node.cpp

```diff
@@ -27,11 +27,39 @@ namespace starrocks::vectorized {
 CrossJoinNode::CrossJoinNode(ObjectPool* pool, const TPlanNode& tnode, const DescriptorTbl& descs)
         : ExecNode(pool, tnode, descs) {}
 
+static bool _support_join_type(TJoinOp::type join_type) {
+    // TODO: support all join types
+    switch (join_type) {
+    case TJoinOp::CROSS_JOIN:
+    case TJoinOp::INNER_JOIN:
+    case TJoinOp::LEFT_OUTER_JOIN:
+    case TJoinOp::RIGHT_OUTER_JOIN:
+    case TJoinOp::FULL_OUTER_JOIN:
```

Comment:
can NL join  support anit join and semi join?

Reply:
No. Currently there's no need for `anti join` and `semi join`.

Reply:
In theory it could support, I could implement it in the future.